src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -1438,4 +1438,56 @@ struct CORJIT_FLAGS
         public CorJitFlag corJitFlags;     // Values are from CorJitFlag
         public CorJitFlag2 corJitFlags2;   // Values are from CorJitFlag2
     };
+
+    //
+    // Constants for fixup signature encoding
+    //
+
+    enum ReadyToRunFixupKind : ushort
```

Comment:
We should create `ReadyToRun.cs` that is C# clone of https://github.com/dotnet/coreclr/blob/master/src/inc/readytorun.h, and move both this and `JitHelperId` (renamed to `ReadyToRunHelper`) to it.